test/unit/search_indexer_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+
+class SearchIndexerTest < ActiveSupport::TestCase
+  def test_indexing_to_rummager
+    artefact = FactoryGirl.create(:artefact, content_id: "content-id")
+    edition = FactoryGirl.create(
+      :guide_edition,
+      title: "A title",
+      overview: "An overview",
+      panopticon_id: artefact.id,
+      primary_topic: "some/topic",
+      additional_topics: %w(foo bar),
+      browse_pages: %w(tax),
+    )
+    registerable_edition = RegisterableEdition.new(edition)
+
+    Services.rummager.expects(:add_document).with(
+      'edition',
+      "/#{edition.slug}",
+      content_id: "content-id",
+      rendering_app: "publisher",
+      publishing_app: "publisher",
+      format: "guide",
+      title: "A title",
+      description: "An overview",
+      indexable_content: "",
```

Comment:
Can you add a test that shows we send something sensible to `indexable_content`?